src/collections/_documentation/performance-monitoring/configuration/python.md

```diff
@@ -140,10 +140,11 @@ endpoint
 
 parameters
 
-: ```{
-"user_id": 314159265358979323846264338327,
-"tracking_id": "EasyAsABC123OrSimpleAsDoReMi",
-"product_name": PlantToHumanTranslator,
-"product_id": 161803398874989484820458683436563811772030917980576,
-}
+: ```python
+    {
+        "user_id": 314159265358979323846264338327,
+        "tracking_id": "EasyAsABC123OrSimpleAsDoReMi",
+        "product_name": PlantToHumanTranslator,
+        "product_id": 161803398874989484820458683436563811772030917980576,
+    }
```

Comment:
```suggestion
: ```json
    {
        "user_id": 314159265358979323846264338327,
        "tracking_id": "EasyAsABC123OrSimpleAsDoReMi",
        "product_name": "PlantToHumanTranslator",
        "product_id": 161803398874989484820458683436563811772030917980576,
    }
```

Tested it this time :)
Also could we update this parameter in the JavaScript example too?

Reply:
Updated for JS, too